docs/content/preview/explore/ysql-language-features/foreign-data-wrappers.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Foreign Data Wrappers
+linkTitle: Foreign Data Wrappers
+description: Foreign Data Wrappers in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-foreign-data-wrappers
+    parent: explore-ysql-language-features
+    weight: 100
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how to use foreign data wrappers.
+
+## Overview
+
```

Comment:
I think you could just eliminate these and start the page with the definition. That'll also let us bump the headings up a level and make the right-side navigation cleaner.
```suggestion
```